File: Python/hangman.py
word = list(input())
guess = list(input())

lives = 10
for i in guess:
    #print(word)

    if(len(word) == 0):
        break

    if(word.__contains__(i)):
        while(word.__contains__(i)):
            word.remove(i)

    else:
        lives = lives -1

if(lives > 0):
    print("WIN")
else:
    print("LOSE")

File: Java/hangman.java
import java.util.Scanner;

public class hangman{
    
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        String word = sc.nextLine();
        String guess = sc.nextLine();
        int lives = 10;
        for(int i = 0; i < guess.length(); i++){
            if(word.length() == 0){
                break;
            }
            
            if(word.contains("" + guess.charAt(i))){
                word = word.replace("" + guess.charAt(i), "");
            }
            else{
                lives --;
            }
        }
    
        if(lives > 0){
            System.out.println("WIN");
        }
        else{
            System.out.println("LOSE");
        }
        
    }
}
